memphis/memphis.py

```diff
@@ -295,13 +400,21 @@ async def produce(self, message, ack_wait_sec=15, headers={}, async_produce=Fals
             Exception: _description_
         """
         try:
+            subject = get_internal_name(self.station_name)
```

Comment:
move it out from the produce function, it will be called for each produce operation